pkg/base/BUILD.bazel

```diff
@@ -64,3 +65,9 @@ go_test(
         "@com_github_stretchr_testify//require",
     ],
 )
+
+# Using common function for stringer to create testclusterreplicationmode_string.go
+stringer(
```

Comment:
Hmm. I don't like that it's unclear what the `name` of this rule is. Can you add a `name` argument to the `stringer()` helper function? Then the call sites would have to be updated as well.

Basically I'm worried that in the future someone will CTRL-F for `gen-testclusterreplicationmode-stringer` and not be able to find it, which is confusing. Also the name `gen-testclusterreplicationmode-stringer` itself is long and silly, you could probably pick a better one by hand.

Reply:
I am using that name `gen-testclusterreplicationmode-stringer` using this logic gen + name of the file + stringer.
that name I think is the longest (so far) but the others are like: Type, Key, etc.

And for the `name` in argument I can add it, but not sure if that change will do any major change (as this is the file that has the longest name)